### ISSUE TITLE
It's not possible to copy slides across tabs

### ISSUE THREAD

**Issue by JMoVS** [CollaboraOnline/online#6836]
**Describe the bug**
When one has two presentations open in 2 tabs, it's not possible to copy a slide on one tab and insert it in another.

**To Reproduce**
1. Open 2 tabs with two separate presentation documents
2. Copy a slide in one tab
3. try to paste it in the other tab

**Expected behavior**
Copy+Paste working across tabs

**Actual behavior**
A weird empty box is inserted

**Screenshots**

**Desktop (please complete the following information)**
 - OS: macOS
 - Browser: Firefox

**Additional context**

This is sorely missing as when works inside a cloud like Nextcloud, sometimes one wants to copy slides from different decks into one - in these cases, one needs this.

**Comment by gitgt** [comment]
True, I mark as Enhancement. Only, this probably will not work via Copy+Paste, rather Ctrl+C, Ctrl+V.  T40445

**Comment by JMoVS** [comment]
for me as a user there is no functional difference between copy and paste and the keyboard shortcuts for copy and paste (on mac cmd-c and cmd-v). I consider it a bug as I would see it as basic functionality (eg onlyoffice has this and it works fine there).

**Comment by gitgt** [comment]
Bug is when something existing does not work. This was never made, it is different from new copy+paste menu in 23.05, and thus this is a new feature, enhancement.

**Comment by timur-g** [comment]
That should be working in 25.04 from https://github.com/CollaboraOnline/online/pull/11659, both using right click context menu or shortcuts.
Later there was change in https://github.com/CollaboraOnline/online/pull/13008  addressed in https://github.com/CollaboraOnline/online/pull/13384
If it does not in some cases where an empty box appears, please open another ticket.
